CoreErrorReportView/src/au/gov/asd/tac/constellation/views/errorreport/ErrorReportTopComponent.java

```diff
@@ -0,0 +1,894 @@
+        finePopupAllowed.setPrefSize(5, 5);
+        finePopupAllowed.setPadding(new Insets(0));
+        
+        severeReportFilter.setPrefSize(20, 3);
+        severeReportFilter.setPadding(new Insets(0));
+        warnReportFilter.setPrefSize(20, 3);
+        warnReportFilter.setPadding(new Insets(0));
+        infoReportFilter.setPrefSize(20, 3);
+        infoReportFilter.setPadding(new Insets(0));
+        fineReportFilter.setPrefSize(20, 3);
+        fineReportFilter.setPadding(new Insets(0));
+
+        settingsBox.setPrefSize(23, 21);
+        settingsBox.setPadding(new Insets(1,0,0,0));
+        settingsBox.addEventFilter(MouseEvent.MOUSE_ENTERED, new EventHandler<MouseEvent>() {
+            public void handle(final MouseEvent mouseEvent) {
+                flashErrorIcon(false);
+                mouseEvent.consume();
+            }
+        });
+        
+        severePopCheckBox.setSelected(true);
+        severePopCheckBox.setOnAction((final ActionEvent event) -> updateSettings());
+        severePopCheckBox.setPadding(new Insets(0, 0, 0, 0));
+        warningPopCheckBox.setSelected(true);
+        warningPopCheckBox.setOnAction((final ActionEvent event) -> updateSettings());
+        warningPopCheckBox.setPadding(new Insets(0, 0, 0, 0));        
+        infoPopCheckBox.setSelected(false);
+        infoPopCheckBox.setOnAction((final ActionEvent event) -> updateSettings());
+        infoPopCheckBox.setPadding(new Insets(0, 0, 0, 0));
+        finePopCheckBox.setSelected(false);
+        finePopCheckBox.setOnAction((final ActionEvent event) -> updateSettings());
+        finePopCheckBox.setPadding(new Insets(0, 0, 8, 0));
+        
+        severeRepCheckBox.setSelected(true);
+        severeRepCheckBox.setOnAction((final ActionEvent event) -> {filterUpdateDate = new Date(); updateSettings();});
+        severeRepCheckBox.setPadding(new Insets(0, 0, 0, 0));
+        warningRepCheckBox.setSelected(true);
+        warningRepCheckBox.setOnAction((final ActionEvent event) -> {filterUpdateDate = new Date(); updateSettings();});
+        warningRepCheckBox.setPadding(new Insets(0, 0, 0, 0));
+        infoRepCheckBox.setSelected(true);
+        infoRepCheckBox.setOnAction((final ActionEvent event) -> {filterUpdateDate = new Date(); updateSettings();});
+        infoRepCheckBox.setPadding(new Insets(0, 0, 0, 0));        
+        fineRepCheckBox.setSelected(true);
+        fineRepCheckBox.setOnAction((final ActionEvent event) -> {filterUpdateDate = new Date(); updateSettings();});
+        fineRepCheckBox.setPadding(new Insets(0, 0, 2, 0));
+
+        updateSettings();        
+        
+        MenuButton filterControl = new MenuButton("Report Settings ");
+        CustomMenuItem severityItem = new CustomMenuItem(severePopCheckBox);
+        severityItem.setHideOnClick(false);
+        filterControl.getItems().add(severityItem);
+        severityItem = new CustomMenuItem(warningPopCheckBox);
```

Comment:
wondering if it would be better to create different variables for each of the items you want to add rather than reusing the same one. Just in case any unexpected behaviour arises from re-use (also allows you refer back to a particular item if need be going forward)